Flow.Launcher.Core/Plugin/PluginManager.cs

```diff
@@ -171,7 +171,7 @@ public static async Task<List<Result>> QueryForPlugin(PluginPair pair, Query que
 
                 token.ThrowIfCancellationRequested();
                 if (results == null)
-                    return results;
```

Comment:
just curious why return null when results is already null? does this actually change anything?